packages/template-retail-react-app/config/default.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
         commerceAPI: {
             proxyPath: `/mobify/proxy/api`,
             parameters: {
-                clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836',
+                clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836', // SLAS Private Client
```

Comment:
Dumb question.. why did you make this call out? Simply to help ourselves out?

Reply:
Ah I think Carson may have added that in for clarity when he was working on this PR. I can revert this!